web/src/lib/components/photos-page/asset-date-group.svelte

```diff
@@ -32,6 +32,8 @@
 	let hoveredDateGroup = '';
 	$: assetsGroupByDate = lodash
 		.chain(assets)
+		.sortBy((a) => a.fileCreatedAt)
+		.reverse()
```

Comment:
Can these be replaced with `orderBy` instead? I think something like this would do the trick:

```typescript
.orderBy(['fileCreatedAt'], ['desc'])
```

Reply:
I'm not quite sure. It definitely behaves different than sortBy and I'm not sure why, but apparently that may be caused by different behaviour between asc and desc when there are identical values.
When I use your example I get the exact same order that is already provided by the asset array so either it does nothing or the array is already sorted and doesn't need to be sorted. If that's the case, we could just remove the sorting from the grid view and the asset viewer altogether. Otherwise we have to decide whether sortBy or orderBy is the way to go because using one function in one module and the other function in the other module definitely doesn't work.